### PR TITLE
chore: add actions to the readme

### PR DESCRIPTION
**Pull request by krol3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@
   - [Routes](#routes)
     - [Route Plugins](#route-plugins)
   - [Templates](#templates)
+  - [Actions](./docs/actions/actions.md#postee-actions)
+    - [Exec Action](./docs/actions/actions.md#exec-action)
+    - [Http Action](./docs/actions/actions.md#http-action)
```
Comment (simar7) [inline README.md:34]:
```suggestion
    - [HTTP Action](./docs/actions/actions.md#http-action)
```

Comment (simar7) [inline README.md:34]:
```suggestion
    - [HTTP Action](./docs/actions/actions.md#http-action)
```

File: docs/actions/actions.md [hunk 2]
```diff
@@ -45,6 +48,8 @@ Below is an example of using `$POSTEE_EVENT`. It uses the inline exec-script scr
 
 As you can see, we capture the incoming Postee event and write this event to the Tracee event log for forensic purposes.
 
+## Http Action
```
Comment (simar7) [inline docs/actions/actions.md:51]:
```suggestion
## HTTP Action
```
```suggestion
## Http Action
```

Comment (simar7) [inline docs/actions/actions.md:51]:
```suggestion
## HTTP Action
```
```suggestion
## Http Action
```

**Comment by simar7** [comment]
thanks this is great! After we merge this, lets start a PR that renames all outputs -> actions as well. We want to go ahead with the direction of making everything an Action, including existing "outputs".

**Review by simar7 — APPROVED** [review]

**Comment by krol3** [comment]
> thanks this is great! After we merge this, lets start a PR that renames all outputs -> actions as well. We want to go ahead with the direction of making everything an Action, including existing "outputs".

Do you mean in the readme, this section: https://github.com/aquasecurity/postee#outputs --> change to action too, doesn't it?

**Pull request merged**